crates/fluvio-cluster/src/cli/delete.rs

```diff
@@ -19,10 +21,38 @@ pub struct DeleteOpt {
     #[arg(long = "sys", conflicts_with = "k8_only")]
     /// Remove system chart only
     sys_only: bool,
+
+    /// Do not prompt for confirmation
+    #[arg(long)]
+    force: bool,
 }
 
 impl DeleteOpt {
     pub async fn process(self) -> Result<()> {
+        if !self.force {
```

Comment:
Can this also output a message if `self.force == true` that `cluster/endpoint` is being deleted.

Reply:
Done!